content/index.html

```diff
@@ -67,7 +67,11 @@ <h2 class="faq"><i class="fas fa-comments"></i></h2>
     </li>
     <li>
         <p class="heading">Do you have plans for supporting iOS apps?</p> 
-        <p>Yes, in the long run, we'd like to be able to run iOS apps on ARM devices (like most Android phones). A significant challenge here would be to write our own implementation of UIKit. Come talk to us if you're interested in working on this!</p>
+        <p>Yes, in the long run, we'd like to be able to run iOS apps on ARM devices (like most Android phones). Come talk to us if you're interested in working on this! In the future, support for tvOS, watchOS, and visionOS software is planned too.</p>
```

Comment:
> In the future, support for tvOS, watchOS, and visionOS software is planned too.

The other Darling members may disagree with me on this, but I don't think we should make promises like this until our macOS (and eventually iPadOS) implementation are in a decent shape.

Reply:
I agree, IMO we should focus on macOS and iOS/iPadOS for now; I personally hadn't even considered those platforms yet.

Reply:
+1, let's not get ahead of ourselves. If someone wants to work on those platforms, sure, but otherwise let's not advertise something that is clearly not there and unlikely to happen.